### DISCUSSION TITLE
new feed sucks

### DISCUSSION THREAD

**Discussion by kaganisildak** [community/community#66229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

wtf's going here?

**Comment by flrngel** [comment]
someone had to work on something because he/she got paid

**Reply by kaganisildak** [reply]
"product manager"
![Alt Text](https://media.giphy.com/media/z0ImSbMh4uJR4BEJju/giphy.gif)

**Reply by kelleyma49** [reply]
No, no, no, no. I do want this. 

Please bring back the tab for showing only the things I'm following. In the new feed, I see a reference to a php repo. I don't know php. I don't care about php. This just adds more noise and junk I have to sift through.

![TheOfficeNoGIF](https://github.com/community/community/assets/2152684/1dd5ea3a-d04d-4a48-839b-246a42b15b71)

**Reply by 0xRapid** [reply]
I want back my old feed, this new feed is #$%J#$K%J#$K%

**Reply by softworkz** [reply]
What ISSSS THIISSSSS???

I need the previous view back. Please!!!!

**Reply by port19x** [reply]
There are so many cool things that could be added to github, features that could be improved and expanded upon, yet we have this crap?
At this point just hire any regular github user as the new product manager.
Sad how out of touch upper management must be at GH right now

**Comment by 0x676e67** [comment]
Today's update sucks. I can't see any project notifications from people following me.

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Comment by lwojcik** [comment]
I used 'following' feed to specifically see what other people I follow are up to. With this change following people becomes nearly pointless because the signal I'm interested in gets lost in noise I don't want.

Every other platform in the EU enables chronological / not personalized feeds (mostly because Digital Market Act forces them to). Meanwhile GitHub: 🤡 

If you really spend time and resources on implementing a social layer in a code sharing platform - at least get it right.

**Comment by eddyizm** [comment]
This is so bad. it is over the top and ridiculous. Need to at least make it easy to keep your feed the way you had it before, all this new garbage is ridic, this is not social media.

**Comment by kaganisildak** [comment]
still sucks

**Comment by GabrielModog** [comment]
still sucks and the pagination isn't working anymore lol

**Comment by mouadziani** [comment]
still sucks

**Comment by kaganisildak** [comment]
damn still sucks

**Comment by savokiss** [comment]
Oh, I can't see any news from the guys I've followed. The new feeds make no sense.
I want the old feeds back!
